events.html

```diff
@@ -0,0 +1,18 @@
+<!DOCTYPE html>
+<html>
+    <head>
+        <title>events lab</title>
+        <script src= "./events.js"></script>
+    </head>
+    <body>
+        <h1>My To-do's</h1>
+        <form>
+            <label for="list">List:</label>
+            <input id="list" type="type" value=></input>
```

Comment:
Value should be equal to something, it can be an empty string, the input tag can be closed like that `<input id="list" type="type" value="" />`

Reply:
I tried this but it does not work with empty string